internal/payout/distribution.go

```diff
@@ -0,0 +1,64 @@
+package payout
+
+import (
+	"github.com/NodeFactoryIo/vedran/internal/models"
+	"math"
+	"math/big"
+)
+
+const (
+	livelinessRewardPercentage = 0.1
+	requestsRewardPercentage   = 0.9
+	roundPrecision             = 1_000_000_000_000_000
+)
+
+func CalculatePayoutDistributionByNode(
+	payoutDetails map[string]models.NodeStatsDetails,
+	totalReward float64,
+	loadBalancerFee float64,
```

Comment:
Maybe it makes more sense to call this loadBalancerFeePercentage to name it similar to livelinessReward and requestReward percentages and then call rewardPool * loadBalancerFeePercentage = loadbalancerReward

Reply:
:+1:

Reply:
:+1: